### PR TITLE
Add Windows slave hosted by Rackspace

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
=====

In this way it should be easier to allow people to access it for maintenance

I saw that we are using only about $700 of rax invoice.

The windows slave was quite busy sometimes so a second slave should help

Changes
=======

I have created the smallest win2012r2 windows vm on RAX

The private repo was already pushed.

I am documenting the steps required to have a windows slave here https://twistedmatrix.com/trac/wiki/ContinuousIntegration/Buildslaves#BuildslaveonWindows

For now the new slaves are in the unsuported category as I did not managed to start buildslave a a service. 

I can start it in debug mode... but when I use the Windows service tool, it fails to start

The builddir option is no longer used as I think that it is automatically created by buildslave based on the builder name

To do
====

* Start buildslave as a service

How to test
=========

Changes are deployed in production

https://buildbot.twistedmatrix.com/builders/win-64-py2.7

https://buildbot.twistedmatrix.com/builders/win-64-py2.7-coverage

### CODE REVIEW

**Comment by adiroiban** [comment]
the bdist builder fails to upload the file... I am still investigating

please check that initial changes are ok.

Thanks!

File: services/buildbot/master/master.cfg [hunk 1]
```diff
@@ -80,6 +80,8 @@ ubuntu16_04_slaves = ['rax-ubuntu1604-%d' % (i,) for i in range(1,4)]
 
 allpy27_slaves = fedora22_slaves + fedora23_slaves + ubuntu12_04_slaves + ubuntu16_04_slaves
 
+win_slaves = ['bot-glyph-6', 'rax-win2012r2-1']
```
Comment (hawkowl) [inline services/buildbot/master/master.cfg]: Doesn't look like this is used anywhere?

File: services/buildbot/master/master.cfg [hunk 2]
```diff
@@ -247,6 +249,51 @@ builders.append({
...
+                      EXTRA_DEPENDENCIES + WINDOWS_DEPENDENCIES),
+        python="C:\\Python27\\python.exe", platform="windows",
+        buildID="windows64py27",
+        uncleanWarnings=True,
+        ),
+    'category': 'unsupported',
+    })
+
+
+builders.append({
```
Comment (hawkowl) [inline services/buildbot/master/master.cfg]: I think this will overwrite the currently uploaded wheels... could we disable this one for now?

**Comment by hawkowl** [comment]
See comments, fix them up and then merge.

File: services/buildbot/master/master.cfg (hunk 1 above)
Comment (adiroiban) [inline services/buildbot/master/master.cfg]: Yep. I will rename it. It was there as a reminder of my intention to unify .

I will remove and use a separate win2012r2 builder

**Comment by adiroiban** [comment]
Thanks. I am adding it as unsupported for now.

It can be used to send coverage reports  for windows in case win7 is down :)

I hope that we can get some VMs from Azure and them move all Windows slaves there.

**Pull request merged**